src/index.js

```diff
@@ -102,6 +104,13 @@ class LndEngine {
     this.retrieveWindowDuration = retrieveWindowDuration || 6000
     this.claimWindowDuration = claimWindowDuration || 30 * this.secondsPerBlock
 
+    // The amount of time, in seconds, that we'd like to buffer any output timelock
```

Comment:
I think we can add a little more explanation: "The recipient requires a minimum number of blocks for the time lock on an inbound HTLC. If we use this exact value to create the HTLC on the sending side, there is a small chance that a block will be mined during the time it takes for the HTLC to be sent over the network to the recipient, and this will cause the recipient to reject the HTLC."